godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -30,6 +67,138 @@ class TractViewController: BaseViewController {
         }
     }
     
+    fileprivate func initializeView() {
+        let navigationBarFrame = navigationController!.navigationBar.frame
+        let startingPoint = navigationBarFrame.origin.y + navigationBarFrame.size.height
```

Comment:
This variable name is a little confusing because it has "Point" in the name, but it's not a `CGPoint`. Could we name it "startingYPos" so it's clear later that we are dealing with the y position only as a `CGFloat`?